guillotina/json/serialize_content.py

```diff
@@ -56,20 +56,24 @@ def __init__(self, context, request):
         else:
             parent_summary = {}
 
+        factory = get_cached_factory(self.context.type_name)
+        behaviors = []
+        for behavior_schema in factory.behaviors or ():
+            behaviors.append(behavior_schema)
+
         result = {
             '@id': IAbsoluteURL(self.context, self.request)(),
             '@type': self.context.type_name,
             '@name': self.context.__name__,
             '@uid': self.context.uuid,
             'parent': parent_summary,
             'is_folderish': IFolder.providedBy(self.context),
+            '__static_behaviors__': behaviors,
```

Comment:
Can we use a `@` value here?

Sorry, I don't like adding non-editable attributes without the `@`.